lib/cli_common/cli_common/pulse.py

```diff
@@ -45,8 +45,11 @@
         connection_global=False
     )
 
-    queue = 'queue/{}/{}'.format(user, exchange)
+    queue = f'queue/{user}/{exchange_name}'
+    exchange = f'exchange/{user}/{exchange_name}'
```

Comment:
In the code coverage case, the exchange should be `exchange/taskcluster-queue/v1/task-group-resolved`. Is adding the user going to make it work?